<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>串口</string>
     </attribute>
     <widget class="QTextEdit" name="send_textedit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>391</width>
        <height>461</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="overwriteMode">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextEditorInteraction</set>
      </property>
     </widget>
     <widget class="QPushButton" name="seachcom">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>430</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索串口</string>
      </property>
     </widget>
     <widget class="QComboBox" name="bandratebox">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>490</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
     <widget class="QPlainTextEdit" name="rec_textedit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>361</width>
        <height>381</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>381</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>361</width>
        <height>381</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QComboBox" name="serialcom">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>460</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>460</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>串口：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>492</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>波特率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>430</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>检验位：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>460</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>数据位：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>490</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>停止位：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="checkbox">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>430</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>None 无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odd 奇</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Even 偶</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="databox">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>460</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>8bit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7bit</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="stopbitbox">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>490</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1bit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2bit</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="opencom">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>440</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>531</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>提示：</string>
      </property>
     </widget>
     <widget class="QLabel" name="errorlabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>530</y>
        <width>731</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>接收：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>发送：</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="hex1button">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>HEX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="str1button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>字符串</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>0</y>
        <width>114</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="hex2button">
         <property name="text">
          <string>HEX </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="str2button">
         <property name="text">
          <string>字符串</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>清空显示</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SendButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>500</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clrsendbutton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>0</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>清空输入</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="autosendcheckBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>500</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>自动发送</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="msBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>500</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>间隔(ms):</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>hex文件读取</string>
     </attribute>
     <widget class="QLabel" name="tab2warnlab">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>540</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>提示：</string>
      </property>
     </widget>
     <widget class="QLabel" name="tab2warn_dis">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>540</y>
        <width>661</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="fileHexEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>381</width>
        <height>471</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>文件：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filenameedit">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="FileOpenButton">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>打开文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPlainTextEdit" name="fileTextEdit">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>371</width>
        <height>471</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>HEX显示：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>字符串显示：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>文件发送</string>
     </attribute>
     <widget class="QTextEdit" name="FileSendEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>771</width>
        <height>201</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>2</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>文件发送数据:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="updataradioButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DRV升级格式</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SendFileButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>500</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>发送文件</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
